src/Umbraco.Infrastructure/Security/MemberUserStore.cs

```diff
@@ -308,24 +363,37 @@ protected override async Task<IdentityUserLogin<string>> FindUserLoginAsync(stri
             cancellationToken.ThrowIfCancellationRequested();
             ThrowIfDisposed();
 
-            MembersIdentityUser user = await FindUserAsync(userId, cancellationToken);
+            MemberIdentityUser user = await FindUserAsync(userId, cancellationToken);
             if (user == null)
             {
-                return null;
+                //TODO: error throw or null result?
```

Comment:
Nulls are good, we'll follow what the entity framework implementation does https://github.com/dotnet/aspnetcore/blob/main/src/Identity/EntityFrameworkCore/src/UserStore.cs#L308